src/DataStructures/Tensor/Python/InstantiateScalar.cpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/Tensor/Python/Tensor.tpp"
```

Comment:
Do you also need the `.hpp` file here as well?

Reply:
It's included in the tpp 🤷‍♂️ this file just does instantiations so I think this is fine

Reply:
Oh this is the scalar file.. wasn't sure here either, but doesn't matter much I think

Reply:
No it doesn't really matter. I thought we might have a guideline for this in our docs, but I couldn't find one. So this seems ok.

Reply:
I think when the tpp+hpp case first came up none of us could figure out what the right answer was, so we just left it out of the guidelines and let people do what they felt like.